packages/mdc-ripple/util.js

```diff
@@ -15,13 +15,21 @@
  */
 
 /** @private {boolean|undefined} */
+let supportsCssVariables_;
+
+/** @private {boolean|undefined} */
 let supportsPassive_;
 
 /**
  * @param {!Window} windowObj
+ * @param {boolean=} forceRefresh
```

Comment:
>  forceRefresh is only really needed for testing purposes

OMG yes, we should def document that its only for testing purpose then! Why is it only for testing purposes??